internal/httputil/headers.go

```diff
@@ -1,11 +1,20 @@
 package httputil // import "github.com/pomerium/pomerium/internal/httputil"
 
+// Pomerium headers contain information added to a request.
 const (
 	// HeaderPomeriumResponse is set when pomerium itself creates a response,
 	// as opposed to the downstream application and can be used to distinguish
 	// between an application error, and a pomerium related error when debugging.
 	// Especially useful when working with single page apps (SPA).
 	HeaderPomeriumResponse = "x-pomerium-intercepted-response"
+	// HeaderPomeriumJWTAssertion is the header key containing JWT signed user details.
+	HeaderPomeriumJWTAssertion = "x-pomerium-jwt-assertion"
+	// HeaderPomeriumUserID is the header key containing the user's id.
+	HeaderPomeriumUserID = "x-pomerium-authenticated-user-id"
```

Comment:
Should we take the steps to just remove the non-JWT metadata headers in this PR, rather than shift it around for a later removal?

Reply:
I’m for it if you are. Note, this would result in email and groups no longer being added to the request logs. We could add ‘jwt’ instead if debug is enabled. Thoughts?

Reply:
I think that makes sense.